test/integration/docker_network_test.go

```diff
@@ -0,0 +1,59 @@
+// +build integration
+
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+)
+
+func TestDockerNetworkFlag(t *testing.T) {
+	if !KicDriver() {
+		t.Skip("only runs with docker driver")
+	}
+
+	tests := []struct {
+		description string
+		flag        string
+	}{
+		{
+			description: "create custom network",
+			flag:        "--docker-network=\"\"",
```

Comment:
it is not clear that what is this test case, 

do we epect if they specify "\" as network name to create a network with that name?
if thats the case we dont verify in the test that we do create that network.

Reply:
if we say "" then it defaults to creating its own network, but i can remove this if it's confusing

Reply:
yeah I think this test be removed,